stdlib/3/collections/__init__.pyi

```diff
@@ -190,7 +190,7 @@ class UserString(Sequence[str]):
 # But in practice it's not worth losing sleep over.
 class deque(MutableSequence[_T], Generic[_T]):
     @property
-    def maxlen(self) -> Optional[int]: ...
+    def maxlen(self) -> Optional[Union[int, None]]: ...
     def __init__(self, iterable: Iterable[_T] = ...,
                  maxlen: int = ...) -> None: ...
```

Comment:
I think you want to add `Optional` here instead. (`Optional[T]` is short for `Union[T, None]`, so the change you already made is redundant.)

Reply:
Thank you, I apologize.  The bug was on 194 not 193.  I'll open a new pull request with the fix.